learn-arai/app/register/page.tsx

```diff
@@ -0,0 +1,142 @@
+'use client';
+
+import Image from 'next/image';
+
+import { FormEvent, useState } from 'react';
+import { FaRegUser } from 'react-icons/fa';
+import { FaPhoneAlt } from 'react-icons/fa';
+import { IoMdKey } from 'react-icons/io';
+import { MdOutlineEmail } from 'react-icons/md';
+
+import Input from '@/components/module/form/input';
+
+import sideLoginPicture from '@/public/login/teaching.jpeg';
+
+import '@/app/register/register.css';
+
+export default function Home() {
+    const [message, setMessage] = useState('');
+    async function submitHandle(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault();
+
+        const formData = new FormData(event.currentTarget);
+        const result = await fetch('http://localhost:3000/auth/sign-up', {
```

Comment:
Use env NEXT_PUBLIC_BACKEND_URL instead of hardcode link (this won't work on production)